writer.go

```diff
@@ -445,6 +445,11 @@ func (p *MediaPlaylist) Encode() *bytes.Buffer {
 				p.buf.WriteString(p.Key.Keyformatversions)
 				p.buf.WriteRune('"')
 			}
+			if p.Key.ID != "" {
+				p.buf.WriteString(",ID=\"")
```

Comment:
This needs to be `KEYID` rather than `ID`

Reply:
I had it as ID because that's the name in the struct. I was following the names as they appear in the struct. Or do you think it would be better to change ID everywhere? 
`type Key struct {
	Method            string
	URI               string
	IV                string
	Keyformat         string
	Keyformatversions string
	ID                string
}`